doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,330 @@
+---
+title: "MultiTech Conduit AEP"
+description: ""
+weight: 8
+---
+
+## Connecting MultiTech Conduit to {{%tts%}}
```

Comment:
Documentation does not start with a heading. See also the documentation guidelines: https://github.com/TheThingsNetwork/lorawan-stack/blob/master/DEVELOPMENT.md#documentation-site